test/boulder-config.json

```diff
@@ -149,7 +149,7 @@
   },
 
   "pa": {
-    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration",
+    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration?readTimeout=800ms&writeTimeout=800ms",
```

Comment:
Not totally thrilled about having all this shoved in a URI, but doing URI construction so that it can be split out seems worse.